elfpy/data/acquire_data.py

```diff
@@ -225,6 +236,7 @@ def main(
                         continue
                 if block_pool_info:
                     postgres.add_pool_infos([block_pool_info], session)
+
```

Comment:
@dpaiton I love this new line, really breaks these code blocks into it's logical components and it makes it much easier to read

Reply:
Sure thing! As a rule of thumb we should be purposeful about using them sparingly & preferentially using line-comments to separate blocks. However, that is a "rule of thumb" -- if anyone feels strongly about a newline then it's fine with me.